protoc_plugin/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 15.0.4
```

Comment:
This should increment the second number, as it is a feature, not a bug-fix.
I also think you have to merge and make this 16.1.0.